.github/workflows/main.yml

```diff
@@ -31,43 +37,66 @@ jobs:
 
     steps:
       - uses: actions/checkout@v1
-      - name: setup (linux)
-        if: startsWith(matrix.os, 'ubuntu')
-        run: |
-          sudo apt install cmake-data
-
-      # - name: setup (mac)
-      #   if: startsWith(matrix.os, 'macos')
-      #   run: |
-      #     brew update
-      #     brew upgrade
-      #     brew install cmake openssl
 
       - name: setup (windows)
         if: startsWith(matrix.os, 'windows')
         run: |
           choco install openssl cmake

```

Comment:
I don't think you should need to install cmake

https://github.com/actions/virtual-environments/blob/master/images/win/Windows2019-Readme.md

has the installed item list for the image.